File: buerguer-pig/AMarotoChivite-BurguerPig/AMarotoChivite-Aemet-Java/src/main/java/Ficheros/Aemet/models/dto/AemetDailyMapToExportXML.java
package Ficheros.Aemet.models.dto;

import Ficheros.Aemet.models.AemetDailyConsultToExport;
import org.simpleframework.xml.ElementMap;
import org.simpleframework.xml.Root;

import java.util.Map;

@Root(name = "informe")
public class AemetDailyMapToExportXML {
    @ElementMap(name = "map", key = "key", entry = "dia_value", attribute = true, inline = true)
    private Map<String, AemetDailyConsultToExport> myMap;

    public Map<String, AemetDailyConsultToExport> getMyMap() {
        return myMap;
    }

    public void setMyMap(Map<String, AemetDailyConsultToExport> myMap) {
        this.myMap = myMap;
    }
}

File: aemet/AMarotoChivite-Aemet/AMarotoChivite-Aemet-Kotlin/src/main/kotlin/Ficheros/Aemet/models/dto/AemetDailyMapToExportXML.kt
package Ficheros.Aemet.models.dto

import Ficheros.Aemet.models.AemetDailyConsultToExport
import org.simpleframework.xml.ElementMap
import org.simpleframework.xml.Root

@Root(name = "informe")
class AemetDailyMapToExportXML {
    @field:ElementMap(name = "map", key = "key", entry = "dia_value", attribute = true, inline = true)
    var myMap: Map<String, AemetDailyConsultToExport>? = null
}